inc/ajax.class.php

```diff
@@ -433,14 +434,16 @@ static function createTabs(
                      }
                   });
                }
-
-               var tabs = ui.tab.parent().children();
-               if (tabs.length > 1) {
-                  var newIndex = tabs.index(ui.tab);
-                  $.get(
-                     '".$CFG_GLPI['root_doc']."/ajax/updatecurrenttab.php',
-                     { itemtype: '".addslashes($type)."', id: '$ID', tab: newIndex }
-                  );
+               // We need to manually set the current tab if the main event was prevented
```

Comment:
I do not understand why this piece of code is still required. If you do not reload the tab, then you should not have to update the current tab value.

Reply:
GLPI keep track of the current tab so the next time you open this item (or another item of the same type) you get back to the same tab.

Reply:
OK, but if you prevent the event, it means you did not load the tab. So either you do not display it (and it shoult not be considered as current), either it is already displayed (and so it is already the current one).

Reply:
No, preventing the event doesn't mean you don't display the tab.
It just means that the HTML of the tab was already fetched by a previous request (= its not the first time you click on this tab) so there is no need to fetch it again.

Reply:
OK, I missed that !

You are on tab X, you display tab Y, then you come back to tab X. It is not loaded, but it has to be stored as current one.

Reply:
I added a comment to expain that, as I am pretty sure that I will not be the only one that miss this point.